internal/gengapic/options.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gengapic
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/googleapis/gapic-generator-go/internal/errors"
+)
+
+type transport int
+
+const (
+	grpc transport = iota
+	rest
+	paramError = "need parameter in format: go-gapic-package=client/import/path;packageName"
```

Comment:
nit: add newline or make its own declaration. Usually nice to have iotas in their own block.

Reply:
A new line after line 28? Done.